### PR TITLE
feat: fancy toggle switch

### PR DESCRIPTION
**Pull request by dnlggr**
Adds a more stylized toggle switch than the default bootstrap one. I adapted the code from [a tutorial](https://medium.com/front-end-weekly/build-a-css-only-toggle-switch-using-tailwindcss-d2739882934).

⚠️ Merge this after #62.

## 📸  Before

https://user-images.githubusercontent.com/10026790/152847768-eb5db0b3-eba6-4463-91dd-568d9157e084.mov

## 📸  After

https://user-images.githubusercontent.com/10026790/152847657-7e14b97e-9ce1-431a-a400-f880df59616b.mov

### CODE REVIEW

**Comment by dnlggr** [comment]
Keeping this a draft until #62 is merged.

File: src/components/CreateWallet.jsx [hunk 1]
```diff
@@ -60,8 +61,8 @@ const WalletCreationForm = ({ createWallet, isCreating }) => {
 const WalletCreationConfirmation = ({ createdWallet, walletConfirmed }) => {
   const [userConfirmed, setUserConfirmed] = useState(false)
 
-  const onSwitch = (e) => {
-    setUserConfirmed(e.target.checked)
+  const onToggle = (toggled) => {
```
Comment (dergigi) [inline src/components/CreateWallet.jsx:64]: NIT: `toggle` -> `isToggled` - see [JavaScript Naming Conventions](https://www.robinwieruch.de/javascript-naming-conventions/)

```suggestion
  const onToggle = (isToggled) => {
```

File: src/components/CreateWallet.jsx [hunk 2]
```diff
@@ -60,8 +61,8 @@ const WalletCreationForm = ({ createWallet, isCreating }) => {
...
+    setUserConfirmed(toggled)
```
Comment (dergigi) [inline src/components/CreateWallet.jsx:65]:
```suggestion
    setUserConfirmed(isToggled)
```

**Review by dergigi — COMMENTED** [review]
Looks good to me, one NIT

File: src/components/CreateWallet.jsx (hunk 1 above)
Comment (dnlggr) [reply src/components/CreateWallet.jsx:64]: 💯

**Comment by dnlggr** [comment]
Resolved some conflicts with master and addressed feedback. Will merge now as this is just a small thing anyway.

**Review by theborakompanioni — APPROVED** [review]
lgtm :rocket:

**Pull request merged**